src/main/components/PerformanceMetrics.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const { Chart, Bar } = require('react-chartjs');
```

Comment:
for consistency lets continue with the es6 Syntax. I believe De-structuring  like below will still work.
import { Chart, Bar } from 'react-chartjs';